src/diff/children.js

```diff
@@ -253,10 +253,8 @@ export function diffChildren(
 	}
 
 	// Set refs only after unmount
-	if (refs) {
-		for (i = 0; i < refs.length; i++) {
-			applyRef(refs[i], refs[++i], refs[++i]);
-		}
+	for (i = 0; i < refs.length; i++) {
+		applyRef(refs[i], refs[++i], refs[++i]);
```

Comment:
Not sure if Terser is doing it already, but we know that `i=0` going into this loop because of L238:

```suggestion
	while (i < refs.length) {
		applyRef(refs[i++], refs[i++], refs[i++]);
```

Reply:
Dang. Had hopes for this one. Increases by 2 bytes on my machine. Here is the terser diff if you are curious:

![image](https://user-images.githubusercontent.com/459878/105133075-e4832800-5aa0-11eb-806d-e4513eec2135.png)

Reply:
I haven't checked in-situ but I think this one is smaller:

```suggestion
	while (refs[++i]) {
		applyRef(refs[i], refs[++i], refs[++i]);
```